.github/actions/spelling/expect/expect.txt

```diff
@@ -162,6 +163,7 @@ BITOPERATION
 bitsavers
 bitset
 BKCOLOR
+bke
```

Comment:
Can we make the spellbot ignore the tests? I honestly do not know how many weird shiz like this I’ll add to the tests in the future.

Reply:
There's an `excludes.txt` file which lists file paths that should be ignored by the spellbot, and it looks like we've used that before to exclude a couple of other test files. I'm not sure about adding _all_ tests, but maybe just the `UtilsTests.cpp` if that's got a lot of weird content.